integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/IntegrajonspunktReceiveImpl.java

```diff
@@ -217,6 +240,20 @@ public EDUCore convertAsicEntrytoEduDocument(byte[] bytes) throws MessageExcepti
         throw new MessageException(StatusMessage.UNABLE_TO_EXTRACT_BEST_EDU);
     }
 
+    public Arkivmelding convertAsicEntryToArkivmelding(byte[] bytes) throws IOException, JAXBException, MessageException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, javax.xml.bind.JAXBException, no.difi.meldingsutveksling.noarkexchange.MessageException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)